go/state/ffistate/ffi_state_test.go

```diff
@@ -37,7 +41,11 @@ func TestAccountsAreInitiallyUnknown(t *testing.T) {
 }
 
 func TestAccountsCanBeCreated(t *testing.T) {
-	runForEachCppConfig(t, func(t *testing.T, state state.State) {
+	runForEachCppConfig(t, func(t *testing.T, state state.State, config state.Configuration) {
```

Comment:
Please remove the `Cpp` in the name of this function.

Reply:
Done!